packages/zenn-cli/src/client/components/articles/show/ArticleHeader.tsx

```diff
@@ -72,9 +72,11 @@ export const ArticleHeader: React.VFC<Props> = ({ article }) => {
             <PropertyRow title="published_at">{publishedAt}</PropertyRow>
           )}
 
-          <PropertyRow title="publication_name">
-            {article.publication_name}
-          </PropertyRow>
+          {article.publication_name && (
+            <PropertyRow title="publication_name">
+              {article.publication_name}
+            </PropertyRow>
+          )}
```

Comment:
細かいですが、`article.publication_name` が空文字列(`""`)の場合、その空文字列が表示されてしまうので、

```suggestion
          {!!article.publication_name && (
            <PropertyRow title="publication_name">
              {article.publication_name}
            </PropertyRow>
          )}
```
のように boolean 型に変換するとより安全だと思います。

参考: https://ja.reactjs.org/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored